querysync/java/com/google/idea/blaze/qsync/query/QuerySummary.java

```diff
@@ -202,8 +202,8 @@ public int index(String s) {
     public Query.StoredLabel indexLabel(Label l) {
       return Query.StoredLabel.newBuilder()
         .setWorkspace(index(l.getWorkspaceName()))
-        .setBuildPackage(index(l.getPackage().toString()))
-        .setName(index(l.getName().toString()))
+        .setBuildPackage(index(l.buildPackage()))
```

Comment:
I searched for `getPackage()` and subsequent call to `toString` that introduces backslashes and found https://github.com/bazelbuild/intellij/blob/a88c6db967cd173fa73aeb76d8c8c7bbd0f8be3d/shared/java/com/google/idea/blaze/common/TargetTree.java#L239

Reply:
I'm not sure this has any adverse effects (FWIW `` passes on Windows), but I will fix this just in case.